codalab/model/bundle_model.py

```diff
@@ -688,40 +686,11 @@ def set_starting_bundle(self, bundle, user_id, worker_id):
 
             return True
 
-    def set_offline_bundle(self, bundle):
```

Comment:
Similarly this moved, got renamed, and had its comment updated. No logic changes.